.gitignore

```diff
@@ -9,3 +9,5 @@ build/
 build-docker/
 emu.config
 compile_commands.json
+trezor_monero_tests{,.log}
+tests/trezor_monero_tests{,.log}
```

Comment:
Might be a good idea to use file-relative location for the download directory, not cwd-relative one, so we don't need both of there here.

Reply:
True that. The script is now cwd independent.